billy-france/src/main/java/com/premiumminds/billy/france/services/export/FRCreditNoteEntryData.java

```diff
@@ -30,10 +30,10 @@
     private final FRInvoiceData reference;
 
     public FRCreditNoteEntryData(ProductData productCode, String description, BigDecimal quantity, BigDecimal taxAmount,
-            BigDecimal unitAmountWithTax, BigDecimal amountWithTax, BigDecimal amountWithoutTax, List<TaxData> taxes,
-            FRInvoiceData reference, String unitOfMeasure) {
-        super(productCode, description, quantity, taxAmount, unitAmountWithTax, amountWithTax, amountWithoutTax, taxes, 
-        		unitOfMeasure);
+		BigDecimal unitAmountWithTax, BigDecimal amountWithTax, BigDecimal amountWithoutTax, List<TaxData> taxes,
+		FRInvoiceData reference, String unitOfMeasure, String exemptionCode, String exemptionReason) {
```

Comment:
if exemption is not applied, this new field come with null value?

Reply:
Yes, that would be the case. The fields are only mandatory in case the tax value for the entry is 0. Any other suggestion on how to handle this?

Reply:
I was thinking maybe if this new parameters should be `Optional` or this classes have 2 constructors, one with the exemption and another without.

What do you think?

Reply:
Will do a change.